### DISCUSSION TITLE
A question about swcPlugins logging

### DISCUSSION THREAD

**Discussion by coder-xiaotian** [swc-project/swc#7880] [Q&A] [answered]
### Summary

I have a swc plugin ["use-client"](https://github.com/coder-xiaotian/swc-useclient), I want to print some info when run this plugin, I used tracing in [my code](https://github.com/coder-xiaotian/swc-useclient/blob/84061744332524728cbf317fd983f2cfcc2b23ef/transform/src/lib.rs#L24C5-L27C39), but it doesn't work when I use in [a demo next.js application](https://github.com/coder-xiaotian/antd-useclient).
I don't know which step was configured incorrectly.
I hope someone can help me. This problem has been blocking me for many days.

### Additional information

```js
next.js version: 13.4.9
os: Mac OS 13.4.9
chrome version: 115.0.5790.170
```

**Comment by coder-xiaotian** [comment]
I tried to run 'cargo test' locally using this [repo](https://github.com/swc-project/plugins), but still haven't printed anything. Is there a detailed document explaining how to print debugging logs in the plugin.

**Reply by kdy1** [reply]
`cargo test` prints nothing if it successes

**Reply by coder-xiaotian** [reply]
Thanks for your reply. I want to print some debug info when `cargo test` or using the plugin in next.js, may I know how to configure it.

**Reply by coder-xiaotian** [reply, marked as answer]
Thanks for your reply. I just found out I can use `println!` in my plugin's code directly, after that it can print log in next.js application.
